src/V3EmitCHeaders.cpp

```diff
@@ -254,8 +263,115 @@ class EmitCHeader final : public EmitCConstInit {
         puts("return !(*this == rhs);\n}\n");
         puts("};\n");
     }
+
+    // `retOrArg` should be prefixed by `&` or suffixed by `.data()` depending on its type
+    void emitPackedMember(const AstNodeDType* dtypep, const std::string& fieldname,
+                          const std::string& offset, bool getfunc, const std::string& retOrArg) {
+        dtypep = dtypep->skipRefp();
+        if (const auto* adtypep = VN_CAST(dtypep, PackArrayDType)) {
+            std::string index = m_names.get("__i");
+            std::string forloop = "for (int " + index + " = 0; " + index + " < "
```

Comment:
```suggestion
            const std::string forloop = "for (int " + index + " = 0; " + index + " < "
```

Please add const to most strings etc below.

Reply:
Fixed